### ISSUE TITLE
Earth Mother pantheon belief

### ISSUE THREAD

**Issue by GrantSP** [Thalassicus/cep-bnw#215]
This might not be our doing but this pantheon belief doesn't appear to be doing as it is described.

It is supposed to give 1 Happiness for EACH Copper resource.
My city has 3 resources nearby so I decided to try it.
Firstly I wasn't getting anything so I assumed the tile has to be improved. Did so, but still nothing.
The Happiness breakdown in the tooltip shows the 4 Happiness from the luxury but nothing in the Policies section. (Don't know why I mentioned Policies only, none of the sub-sections show anything)

**Comment by stackpoint** [comment]
That extra happiness is probably handled by the YieldLibrary but I haven't done any work to reactivate it. The extra happiness from Stories are also broken.

**Comment by GrantSP** [comment]
So is this a case of the function change in TopPanel?

**Comment by stackpoint** [comment]
No, there's no happiness at all.

**Comment by GrantSP** [comment]
So it's because the current TopPanel.lua isn't pulling the value from the Yields table?
It's still using:

```
local iHappiness = activePlayer:GetExcessHappiness()
```

not the modded:

```
local iHappiness = Game.Round(activePlayer:GetYieldRate(YieldTypes.YIELD_HAPPINESS_NATIONAL));
```

Is this 2nd code added as a comment in the file by you or @Thalassicus ?

**Comment by stackpoint** [comment]
There is no (active) infrastructure in the YieldLibrary for the TopPanel to get happiness from the modded sources. It's all commented out.

**Comment by GrantSP** [comment]
Sorry for the hassle, I have just realized I have completely misread 2 different beliefs and confused the two of them. This belief is not supposed to give Happiness per resource but Faith.
Doh!

On a sidenote, I have added in that line above and the Happiness tooltip now shows an extra source of Happiness being gained, although it isn't added to the total.

Additionally the CityView breakdown of yields incorrectly shows Culture coming from Policy as coming from Terrain. I'm pretty sure, 86.79% ;) we had that showing as from Policy before. Perhaps this is a result of an older file being used as the base to work from?

Anyway, don't bother responding in this issue. I'm going to close it.
